### ISSUE TITLE
Conflicting field types in your data—warning msg

### ISSUE THREAD

**Issue by heyalbert** [fabien0102/gatsby-starter#68]
Hi,
I set up a new project by using the gatsby-starter. Everything was generated by the starter.
But:
After running `gatsby develop`I'm always getting this warning:
```
warning There are conflicting field types in your data. GraphQL schema will omit those fields.
SitePluginConnection.packageJson.author:
```
Does someone know why? What happened?

**Comment by fabien0102** [comment]
Hi,
regarding your error, I guess that you have something "weird" into your package.json:author.

@heyalbert can you paste your `package.json` here, to have more information?

**Comment by SoraYama** [comment]
The same issue came to me, and things ganna be fine when I comment [these lines](https://github.com/fabien0102/gatsby-starter/blob/master/gatsby-node.js#L20-L30). I'm not sure whether these code is still work properly 🤣

**Comment by fabien0102** [comment]
Should be fixed in the 2.0 ;)
